release_docs/RELEASE.txt

```diff
@@ -742,6 +742,11 @@ Bug Fixes since HDF5-2.0.0 release
 
     Configuration
     -------------
+    - Allow to set postfix of target file names for debug builds
```

Comment:
Remove default setting of CMAKE_DEBUG_POSTFIX if not set.

Move the default setting of CMAKE_DEBUG_POSTFIX to the cacheinit,cmake file usually used by testing. If CMAKE_DEBUG_POSTFIX is not set with a -D option then CMAKE_DEBUG_POSTFIX will be the default provided by CMake itself,